src/main/java/de/tum/in/www1/artemis/service/UserService.java

```diff
@@ -394,6 +392,28 @@ public void removeNotActivatedUsers() {
         return authorityRepository.findAll().stream().map(Authority::getName).collect(Collectors.toList());
     }
 
+    /**
+     * Given a Course object, it returns the number of users enrolled in the course
+     *
+     * @param course - the course object we are interested in
+     * @return the number of students for that course
+     */
+    public long countNumberOfStudentsForCourse(Course course) {
```

Comment:
I put them here because it works over the `User` data model, not the `Course` one, but I am open for discussion :)

Reply:
my thought was to add it to the course service because the only property that is accessed in there is the course object. so the input of the function could be removed.

Reply:
Okay, I moved them, but I do not understand how I am supposed to remove the argument :thinking:

Reply:
Neither do I. I think we can resolve this comment.